bucket/netbeans.json

```diff
@@ -28,9 +28,10 @@
         "re": "Apache NetBeans ([\\d.]+)</a></li>"
     },
     "autoupdate": {
-        "url": "https://www-eu.apache.org/dist/incubator/netbeans/incubating-netbeans/incubating-$version/incubating-netbeans-$version-bin.zip",
+        "url": "https://www.apache.org/dist/incubator/netbeans/incubating-netbeans/incubating-$version/incubating-netbeans-$version-bin.zip",
         "hash": {
-            "url": "$url.sha512"
+            "url": "$url.sha512",
+            "find": "^($sha512)"
```

Comment:
`"find": "^($sha512)"` is not required, scoop knows the file format and extracts the checksum automatically.